### PR TITLE
Bug 1261612 - Mark homescreen as having transparent background

### PR DESCRIPTION
**Pull request by kanru**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e96f40a7c45ec7ccae21202dc522a2eac370a539

File: apps/system/js/browser_frame.js [hunk 1]
```diff
@@ -111,6 +111,7 @@
        * we add this frame to the document, we can't change its app type.
        */
       iframe.setAttribute('mozapptype', 'homescreen');
+      iframe.setAttribute('transparent', 'true');
```
Comment (lissyx) [inline apps/system/js/browser_frame.js]: We need that for search and keyboard also.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=99ffd84d9a830aa672a51db62987e4617abd7f93

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a713e44bc7eedd038b465e297adf868260d67a75

**Pull request merged**